charts/tempo-distributed/values.yaml

```diff
@@ -316,16 +342,42 @@ config: |
         store: memberlist
   distributor:
     receivers:
+      {{ if .Values.traces.jaeger.enabled }}
```

Comment:
can this just be an OR'ed combination of `.Values.traces.jaeger.grpc`, `.Values.traces.jaeger.thriftBinary`, etc.?

Reply:
Done.